crates/json-abi/src/abi.rs

```diff
@@ -246,6 +253,40 @@ impl JsonAbi {
         }
     }
 
+    /// Goes over all structs referenced as internal types in the ABI and changes them to global
+    /// structs.
+    ///
+    /// Namespace for global struct names is managed and in cases of duplicates structs are renamed.
+    pub fn rename_internal_structs(&mut self) {
```

Comment:
I don't think any of this should be exposed publicly, it should only be a sol! implentation detail